pkg/apk/implementation.go

```diff
@@ -79,16 +77,14 @@ func New(options ...Option) (*APK, error) {
 			return nil, err
 		}
 	}
-	rhttp := retryablehttp.NewClient()
-	rhttp.Logger = hclog.Default()
 
 	if opt.fs == nil {
 		// This is expensive so we only want to do it if we aren't passed WithFS.
 		opt.fs = apkfs.DirFS("/")
 	}
 
 	return &APK{
-		client:             rhttp.StandardClient(),
+		client:             http.DefaultClient,
```

Comment:
should we do 

```
httpClient := &http.Client{}
return &APK{
		client:             rhttp.StandardClient(),
		client:             httpClient,
...
```

Reply:
I'm not sure that's necessary, the default client is just `&http.Client{}` and having it named that way removes some ambiguity.

I don't feel strongly though, if you do.